### PR TITLE
Fix onboarding views transitions

### PR DESCRIPTION
**Pull request by jlsilva1994**
this MR fix the issue #45, on back pressed error , removed the finish() call when going next activity solves it!

### CODE REVIEW

File: app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard3LocationActivity.java [hunk 1]
```diff
@@ -57,7 +58,19 @@ protected void onCreate(Bundle savedInstanceState) {
         homeplaceView.onCreate(this.savedInstanceState);
     }
 
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        AppOpsManager appOps = (AppOpsManager) this.getSystemService(this.APP_OPS_SERVICE);
```
Comment (djcouto) [inline app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard3LocationActivity.java]: Explain to me why is this needed. Is there a scenario where you get to `onboard3` without grant the special permisson? :thinking:

File: app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard2SpecialPermissionActivity.java [hunk 2]
```diff
@@ -38,6 +38,5 @@ public void showUsageSettings(View v){
 
     public void goNext() {
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard2SpecialPermissionActivity.java]: Remove this method and inline the only line in it in the code. Calling a method for just one instruction isn't a good idea :+1:

File: app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard1WelcomeActivity.java [hunk 3]
```diff
@@ -34,7 +34,6 @@ protected void onCreate(Bundle savedInstanceState) {
     public void start(View v) {
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard1WelcomeActivity.java:45]: Same as `goNext()` below. I think you can inline this :wink:

**Review by hgg — CHANGES_REQUESTED** [review]
Just some minor changes and we can merge this patch :+1:

File: app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard1WelcomeActivity.java (hunk 3 above)
Comment (jlsilva1994) [reply app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard1WelcomeActivity.java:45]: here i can't inline the method's lines because this is a callback method for the button next. if you check the xml, the button onclick property calls this method

**Review by djcouto — CHANGES_REQUESTED** [review]

**Comment by djcouto** [comment]
@jlsilva1994 if you are doing this on this stages, you should also do it on all onboarding stages, don't you agree? :smile:

**Comment by jlsilva1994** [comment]
i override some onBackPressed methods to do the logic of going back on onboarding, this includes going back one context in the select context apps view

**Review by djcouto — APPROVED** [review]
